storage_tiering.hpp

```diff
@@ -121,6 +121,9 @@ namespace irods {
             rcComm_t*          _comm,
             const std::string& _resource_name);
 
+        auto get_resource_group_tier(RcComm* _comm, const std::string& _resource_name, const std::string& _group_name)
```

Comment:
Hmm... maybe this should be `get_group_tier_for_resource`?

I think there may be other functions laying around that do some superset of this and we could do some in-memory manipulation of that data to figure this out, but it wasn't clear to me whether that was the case (see `get_resource_map_for_group`).

Reply:
`get_group_tier_for_resource` seems gooder.

Reply:
Done